storage2/src/snapshot.rs

```diff
@@ -28,3 +37,50 @@ impl Snapshot {
         self.jmt_version
     }
 }
+
+/// A reader interface for rocksdb. NOTE: it is up to the caller to ensure consistency between the
+/// rocksdb::DB handle and any write batches that may be applied through the writer interface.
+impl TreeReader for Snapshot {
+    /// Gets node given a node key. Returns `None` if the node does not exist.
+    fn get_node_option(&self, node_key: &NodeKey) -> Result<Option<Node>> {
+        let node_key = node_key.clone();
+
+        let span = Span::current();
```

Comment:
This span isn't used. Should it be? What did the previous code do with this (I assume that's why it's here now?)?

Reply:
Another Tokio task artifact